core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/NativeImageBuildLocalContainerRunner.java

```diff
@@ -116,8 +116,14 @@ protected List<String> getContainerRuntimeBuildArgs() {
             volumeOutputPath = FileUtil.translateToVolumePath(volumeOutputPath);
         }
 
+        String selinuxBindOption = ":z";
+        if (SystemUtils.IS_OS_MAC
+                && ContainerRuntimeUtil.detectContainerRuntime() == ContainerRuntimeUtil.ContainerRuntime.PODMAN) {
```

Comment:
Question: will it work with Docker? I would expect it to fail on macOS in both cases?

Reply:
I will test it with MacOS and Docker, and report back.

Reply:
I tested this with Quarkus 2.14.3 on another MacOS with Docker and it worked without any issues. So, this looks to be affecting only Podman.

Another machine details:
MacOS version: 12.6
Docker version: 20.10.12